src/main/java/io/vertx/core/impl/ContextImpl.java

```diff
@@ -221,6 +221,10 @@ public EventLoop getEventLoop() {
           }
         } catch (Throwable e) {
           res.fail(e);
+        } finally {
+          if (blockingCodeHandler != null) {
```

Comment:
I'm thinking that this block is actually not necessary - we don't explicit unset context in other cases, only at close so I think that should be fine in this case too (?)